src/miner.app.src

```diff
@@ -28,6 +28,8 @@
     jsx,
     kvc,
     longfi,
+    public_key,
```

Comment:
I suspect public_key is not necessary, unless erltss2 is not properly declaring its own deps.